cmd/txtar-savedir/testdata/txtar-savedir-self.txt

```diff
@@ -4,7 +4,7 @@
 # stdout check
 txtar-savedir blah
 ! stderr .+
-stdout '^module example.com/blah$'
+stdout '^module example.com[/\\]blah$'
```

Comment:
Can module paths (import paths) contain backslashes?

Reply:
Thanks, I suspect this was the result of an overly aggressive `sed` earlier in refactoring.